index.html

```diff
@@ -8,13 +8,15 @@
     <link rel="stylesheet" href="style.css">
 </head>
 <body>
-    <h1>Awesome Books</h1>
+    <h1>All awesome books</h1>
     <section>
-        <ul id="books-section">
+        <ul id="books-section" class="books-section">
             
         </ul>
```

Comment:
- [ ] The interface currently, is taking the full width of the browser.  Kindly adjust your style for this wrapper and not spread the full width of the browser.